includes/Checker/Checks/Abstract_File_Check.php

```diff
@@ -227,6 +227,15 @@ private static function get_files( Check_Context $plugin ) {
 					}
 				}
 
+				$files_to_ignore = Plugin_Request_Utility::get_files_to_ignore();
+
+				foreach ( $files_to_ignore as $ignore_file ) {
+					if ( str_ends_with( $file, $ignore_file ) ) {
```

Comment:
```suggestion
					if ( str_ends_with( $file, "/$ignore_file" ) ) {
```

Otherwise `--exclude-files=bar.php` will also exclude `foobar.php`, from what I can see here. That should not happen.

Adding a test for such a scenario would be good.